features/consent/src/test/java/in/projecteka/jataayu/consent/viewmodel/UserVerificationViewModelTest.kt

```diff
@@ -98,6 +108,25 @@ class UserVerificationViewModelTest {
         Mockito.verify(call).enqueue(any())
     }
 
+    @Test
+    fun shouldReturnAccountLockWithExceedLimit(){
```

Comment:
can you use back tick and write method like a sentence. so test easily readbale

Reply:
updated